proprietary/tokens/src/components/utrecht/textbox.tokens.json

```diff
@@ -0,0 +1,72 @@
+{
+  "utrecht": {
+    "textbox": {
+      "background-color": {},
+      "border-bottom-width": {},
+      "border-color": {
+        "value": "{amsterdam.color.primary-black}"
+      },
+      "border-radius": {},
+      "border-width": {},
+      "color": {
+        "value": "{amsterdam.color.primary-black}"
+      },
+      "font-family": {
+        "value": "{amsterdam.typography.font-family}"
+      },
+      "font-size": {
+        "value": "{amsterdam.typography.font-size.24}"
+      },
+      "line-height": {},
+      "max-inline-size": {},
+      "padding-block-end": {
+        "value": "{amsterdam.spacing.inset.sm}"
+      },
+      "padding-block-start": {
+        "value": "{amsterdam.spacing.inset.sm}"
+      },
+      "padding-inline-end": {
+        "value": "{amsterdam.spacing.inset.lg}"
+      },
+      "padding-inline-start": {
+        "value": "{amsterdam.spacing.inset.lg}"
+      },
+      "placeholder": {
+        "color": {
+          "value": "{amsterdam.color.neutral-grey3}"
+        }
+      },
+      "disabled": {
+        "background-color": {},
+        "border-color": {
+          "value": "{amsterdam.color.neutral-grey2}"
+        },
+        "color": {
+          "value": "{amsterdam.color.neutral-grey2}"
+        }
+      },
+      "focus": {
```

Comment:
All interactive elements should get a `outline-offset: 2px` (from the `common` tokens). I'll add that to the documentation